esphome/components/gp8403/gp8403.h

```diff
@@ -16,8 +16,9 @@ class GP8403 : public Component, public i2c::I2CDevice {
   void setup() override;
   void dump_config() override;
   float get_setup_priority() const override { return setup_priority::DATA; }
-
+  void set_gp8413(bool gp8413) { gp8413_ = gp8413; }
   void set_voltage(gp8403::GP8403Voltage voltage) { this->voltage_ = voltage; }
+  bool gp8413_;
```

Comment:
```suggestion
  bool gp8413_{false};
```
If `cv.Optional(CONF_GP8413, default=False): cv.boolean,` is optional, then that MUST be initialized in cpp code.